mongo/check.py

```diff
@@ -906,6 +920,13 @@ def total_seconds(td):
                     self._resolve_metric(metric_name, metrics_to_collect)
                 submit_method(self, metric_name_alias, val, tags=metrics_tags)
 
+        if _is_affirmative(instance.get('collections_indexes_stats')):
+            mongo_version = cli.server_info().get('version', '0.0')
+            if LooseVersion("3.2") >= LooseVersion(mongo_version):
```

Comment:
Should this be >**=** ? If mongo_version == 3.2, don't we want it to collect the indexes stats?

Also, it might be more readable if the logic was
```
if LooseVersion(mongo_version) >= LooseVersion("3.2"):
    self._collect_indexes_stats(instance, db, tags)
else:
    self.log.error("'collections_indexes_stats' is only available starting from mongo 3.2: your mongo version is %s", mongo_version)
```